### PR TITLE
Improve favicons

### LINKED ISSUE #22: Update favicons
We seem to be using Vercel's

### PR DESCRIPTION
**Pull request by raulrpearson**
I created a black over white SVG version of the logo shared by Neal and loosely followed [this guide](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs) to decide what files to include. Using [_document.tsx](https://nextjs.org/docs/advanced-features/custom-document) since these are elements that remain constant throughout the app. Closes #22.

### CODE REVIEW

File: packages/frontend/pages/_document.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Head, Html, Main, NextScript } from 'next/document'
+
+const Document = () => {
+  return (
+    <Html>
+      <link rel="icon" href="/favicon.ico" sizes="any" />
```
Comment (wottpal) [inline packages/frontend/pages/_document.tsx:6]: I've never done this in `_document.tsx` (always in `_app.tsx` inside a `<Head>` wrapper), so this question might be dumb: But why is this placed not inside `<Head>…</Head>` but instead before a self-closing `<Head />`?

**Review by wottpal — COMMENTED** [review]
Hey Raul, thanks for taking care of this. I always generate favicons via https://realfavicongenerator.net/ and I can't recommend it more (takes so much hassle away and created copy'n'paste-ready embed-code) but I think you also did it correctly the manual way.

One additional thing to discuss maybe: I always prefer to put the favicons in a subdirectory (i.e. `public/favicons/`) to not clutter the public-directory in its root. With the correct meta/link tags the only browser that does not support this is Internet Explorer and some weird Russian search-indexers afaik. And IE doesn't support crypto wallets either, I guess 😅. I never had issues with that but I understand if you prefer being a good citizen.

File: packages/frontend/pages/_document.tsx (hunk 1 above)
Comment (raulrpearson) [reply packages/frontend/pages/_document.tsx:6]: As discussed, I must have totally spaced out here and didn't realize I wasn't placing them as children to the `Head` component. Thanks for spotting this!

**Comment by raulrpearson** [comment]
@wottpal, I moved all images into the `/favicons` folder and correctly placed all link tags as children of the `Head` component. Feel free to merge if you think everything's good.

**Review by wottpal — APPROVED** [review]
seems good now!

**Pull request merged**